src/main/kotlin/com/demonwav/mcdev/platform/mixin/action/GenerateAccessorAction.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2020 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.platform.mixin.action
+
+import com.intellij.codeInsight.FileModificationService
+import com.intellij.codeInsight.generation.actions.BaseGenerateAction
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.command.CommandProcessor
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.editor.EditorModificationUtil
+import com.intellij.openapi.editor.actionSystem.DocCommandGroupId
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiFile
+import com.intellij.psi.PsiJavaFile
+import com.intellij.psi.util.PsiUtilBase
+
+class GenerateAccessorAction : BaseGenerateAction(GenerateAccessorHandler()) {
+    /**
+     * Copied from [com.intellij.codeInsight.actions.CodeInsightAction.actionPerformedImpl]
+     * except that it calls the [GenerateAccessorHandler.customInvoke] method instead of the normal one
+     */
+    override fun actionPerformedImpl(project: Project, editor: Editor?) {
+        if (editor == null) {
+            return
+        }
+        val psiFile = PsiUtilBase.getPsiFileInEditor(editor, project) ?: return
+        val handler = handler as GenerateAccessorHandler
```

Comment:
I think `val handler = this.handler as GenerateAccessorHandler` would be better here.

If you get around to updating this PR you can make this change, but I'm mostly just marking this as a note for myself when I merge this.